.github/workflows/ios.yml

```diff
@@ -0,0 +1,43 @@
+name: iOS
```

Comment:
This cross compiles for iOS because it builds with the iOS toolchain. That means that it builds an ARM binary. I don't think we should combine them since macos pipeline builds an Intel binary